core/core/scan.go

```diff
@@ -125,6 +125,7 @@ func GetOutputPrinters(scanInfo *cautils.ScanInfo, ctx context.Context) []printe
 
 func (ks *Kubescape) Scan(ctx context.Context, scanInfo *cautils.ScanInfo) (*resultshandling.ResultsHandler, error) {
 	ctxInit, spanInit := otel.Tracer("").Start(ctx, "initialization")
+	logger.InitLogger("icon")
```

Comment:
Use the logger name from the const in the package

Reply:
@dwertent are you referring to importing the entire `iconlogger` package to get that? Or to get it from `ListLoggerNames()[1]`?

Reply:
Import the `iconlogger` pkg so you can use the logger name

Reply:
@dwertent made the change! Thanks!